### ISSUE TITLE
Pregunta de funcionamiento

### ISSUE THREAD

**Issue by RicardoOrtizG** [SIB-Colombia/CESP-GBIF-SiB-VertNet#37]
A partir de los errores que hemos encontrado, nos surgió una duda del funcionamiento del migrador. ¿Hay un punto en proceso en el que el migrador corrobore los nombre científicos contra el backbone taxonómico de GBIF u otros?. Si lo hace, el migrador le da prioridad al master sobre el backbone taxonómico?.

**Comment by tucotuco** [comment]
No, el migrador usa su classificación interna unicamente. Es porque en vertebrados, por lo menos, no estamos de acuerdo con el GBIF Backbone en muchos casos.
